recipes/asap/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "asap" %}
+{% set version = "3.10.0" %}
+{% set sha256 = "d58105290029385835569e1e7057d7b03bad1dd0c5efc2151370e77f7721f478" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: asap-ASAP_VERSION_3_10_0-a78278070c997b302c8117d8174c8914d5f3d273.tar.gz
```

Comment:
Really? Shouldn't. This only sets the name the file is saved under. The checksum should be invariant to that.